### DISCUSSION TITLE
DynJS.axd in serenity

### DISCUSSION THREAD

**Discussion by armanmahmoudi** [serenity-is/Serenity#7133] [Help]
I ran a project and I've wanted to write a lookup to return Role info, but after running it, I saw that all the information of the Roles are in a file named "https://localhost:1429/DynJS.axd/Lookup.Administration.Role.js" which is displayed before the login Why has serenity use DynJS.axd? is there special reason? and finally how to disable this files in my project?

**Comment by VictorTomaili** [comment]
They serve as lookups only, providing the ID and text value of the roles. These are necessary to modify the UI based on role and permissions in certain cases. However, it is still essential to verify the role and permission on the server side.

**Reply by armanmahmoudi** [reply]
But I have set the permission on "?" on server side but still before login I can see the following 
![Capture](https://github.com/serenity-is/Serenity/assets/30973163/0cccb8df-c5c3-4ebf-bf6b-28e273324a86)

**Reply by VictorTomaili** [reply]
You probably have misconfiguration
![image](https://github.com/serenity-is/Serenity/assets/2023440/512bd39c-d6c2-451e-96fe-30503d482d4d)
